client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -37,12 +37,22 @@ class DesignTypeWithStoreStep extends Component {
 	getChoices() {
 		const { translate } = this.props;
 
-		return [
-			{ type: 'blog', label: translate( 'A list of my latest posts' ), image: <BlogImage /> },
-			{ type: 'page', label: translate( 'A welcome page for my site' ), image: <PageImage /> },
-			{ type: 'grid', label: translate( 'A grid of my latest posts' ), image: <GridImage /> },
-			{ type: 'store', label: translate( 'An online store' ), image: <StoreImage /> },
-		];
+		switch ( abtest( 'signupCopy' ) ) {
```

Comment:
I agree with you. I kept getting errors when I used an if/else statement so I decided to replicate an existing pattern on the page.